src/rqt_graph/ros_graph.py

```diff
@@ -304,6 +304,7 @@ def _generate_dotcode(self):
         hide_tf_nodes = self._widget.hide_tf_nodes_check_box.isChecked()
         group_image_nodes = self._widget.group_image_check_box.isChecked()
         hide_dynamic_reconfigure = self._widget.hide_dynamic_reconfigure_check_box.isChecked()
+        simplify = (graph_mode != NODE_NODE_GRAPH)
```

Comment:
I am confident that `NODE_NODE_GRAPH` should have simplification disabled.  However, it could also make sense to disable simplification for all graph types.  If so, we can simply hardcode this as `simplification = False`